ledger/ledger-api-test-tool/src/main/scala/com/daml/ledger/api/testtool/infrastructure/participant/ParticipantTestContext.scala

```diff
@@ -538,6 +554,20 @@ private[testtool] final class ParticipantTestContext private[participant] (
       ),
     )
 
+  def submitRequest(actAs: List[Party], readAs: List[Party], commands: Command*): SubmitRequest =
+    new SubmitRequest(
+      Some(
+        new Commands(
+          ledgerId = ledgerId,
+          applicationId = applicationId,
+          commandId = nextCommandId(),
+          actAs = actAs.map(_.unwrap),
+          readAs = readAs.map(_.unwrap),
```

Comment:
Consider using `.widen` instead of `.map(_.unwrap)` to avoid creating a new list.

Reply:
Can I use `.widen` when the type of element is changing? In this example `.unwrap` maps from `Party` to `String`. `.widen` would change the collection type but not the element type. Am I correct?

Reply:
Use `Party.unsubst` on the list; `widen` will not work. `map` is definitely unnecessary.

However in general @kamil-da `widen` _does_ change the element type. It's just the wrong tool for this particular situation.

Reply:
I haven't noticed the subtype relation between `Party` and `String`. Thank you @S11001001 !

Reply:
Fixed.